docs/releases/4.1.txt

```diff
@@ -26,6 +26,23 @@ officially support the latest release of each series.
 What's new in Django 4.1
 ========================
 
+Define async handlers for class-based views
+-------------------------------------------
+
+View subclasses may now define async HTTP method handlers::
```

Comment:
```suggestion
View subclasses now allows defining async HTTP method handlers::
```

Reply:
Not sure about this one — would be `allow` to match the noun count, but I think I'd still prefer `may now define`, or `can now define` if you don't like `may`, but … 🤔

Reply:
The current version is fine :+1: